manifests/03_configmap.yaml

```diff
@@ -7,3 +7,23 @@ data:
   operator-config.yaml: |
     apiVersion: operator.openshift.io/v1alpha1
     kind: GenericOperatorConfig
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-authentication-operator
+  name: trusted-ca-bundle
+  annotations:
+    release.openshift.io/create-only: "true"
+  labels:
+    config.openshift.io/inject-trusted-cabundle: "true"
```

Comment:
won't CVO stomp this configmap after network operator added the CA ?

Reply:
same below of course

Reply:
@danehans ^^

Reply:
presumably not, because of the `create-only` annotation...

Reply:
nvmd. The create-only label will make sure it is not overwritten.